rct229/rules/section21/section21rule5.py

```diff
@@ -0,0 +1,161 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.ruleset_functions.baseline_systems.baseline_system_util import HVAC_SYS
+from rct229.ruleset_functions.get_baseline_system_types import get_baseline_system_types
+from rct229.ruleset_functions.get_hw_loop_zone_list_w_area_dict import (
+    get_hw_loop_zone_list_w_area,
+)
+from rct229.ruleset_functions.get_zone_conditioning_category_dict import (
+    ZoneConditioningCategory as ZCC,
+)
+from rct229.ruleset_functions.get_zone_conditioning_category_dict import (
+    get_zone_conditioning_category_dict,
+)
+from rct229.schema.config import ureg
+from rct229.utils.assertions import getattr_
+from rct229.utils.jsonpath_utils import find_all, find_exactly_one_with_field_value
+from rct229.utils.pint_utils import ZERO, pint_sum
+
+APPLICABLE_SYS_TYPES = [
+    HVAC_SYS.SYS_1,
+    HVAC_SYS.SYS_5,
+    HVAC_SYS.SYS_7,
+    HVAC_SYS.SYS_11_2,
+    HVAC_SYS.SYS_12,
+    HVAC_SYS.SYS_1A,
+    HVAC_SYS.SYS_7A,
+    HVAC_SYS.SYS_11_2A,
+    HVAC_SYS.SYS_12A,
+]
+
+FLUID_LOOP = schema_enums["FluidLoopOptions"]
+HEATING_LOOP_CONDITIONED_AREA_THRESHOLD = 15000 * ureg("ft2")
+
+
+class Section21Rule5(RuleDefinitionListIndexedBase):
+    """Rule 5 of ASHRAE 90.1-2019 Appendix G Section 21 (Hot water loop)"""
+
+    def __init__(self):
+        super(Section21Rule5, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, False),
+            each_rule=Section21Rule5.RulesetModelInstanceRule(),
+            index_rmr="baseline",
+            id="21-5",
+            description="The baseline building design boiler plant shall be modeled as having a single boiler if the baseline building design plant serves a conditioned floor area of 15,000sq.ft. or less, and as having two equally sized boilers for plants serving more than 15,000sq.ft.",
+            list_path="ruleset_model_instances[*]",
```

Comment:
Elsewhere we have assumed that there is only one RuleModelInstance.
If we assume that here, you can set `list_path="ruleset_model_instances[0]"`. It will then provide a list with one element to the nested rule.

Better yet, if we are assuming only one RuleModelInstance,  the RuleModelInstanceRule could be eliminated and all logic could be performed at the top level.

Reply:
The plan for the ruleset_model_isntances is not entirely clear. I will keep the list_path = 'ruleset_model_instances[0]' for now